src/Text/Pandoc/Writers/Docx.hs

```diff
@@ -1119,6 +1150,9 @@ formattedRun els = do
 setFirstPara :: PandocMonad m => WS m ()
 setFirstPara =  modify $ \s -> s { stFirstPara = True }
 
+setInTable :: PandocMonad m => WS m ()
+setInTable =  modify $ \s -> s { stInTable = True }
+
```

Comment:
A bit awkward to have `setInTable` but then have to use `modify` directly to reset.
Either have `set` and `reset`, or just use `modify`, I'd say.

Reply:
Thanks. I'll switch to `modify` -- I was following the example of `setFirstPara`, which in one case has `modify $ \s -> s { stFirstPara = False }` afterwards.

Reply:
Ah, I see. I guess i'd have the same reaction to that!  (It may be that I'm responsible for it.)